assertj-guava/src/test/java/org/assertj/guava/api/MultimapAssert_doesNotContainKeys_Test.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2024 the original author or authors.
+ */
+package org.assertj.guava.api;
+
+import com.google.common.collect.LinkedHashMultimap;
+import org.junit.jupiter.api.Test;
+
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+import static org.assertj.guava.api.Assertions.assertThat;
+
+public class MultimapAssert_doesNotContainKeys_Test extends MultimapAssertBaseTest {
```

Comment:
the windows build fails because you use `\n` instead of `%n` in the expected error message, just make the change and it should work since `%n` is correctly interpreted by `String.format`